GherkinSyncTool.Synchronizers.AzureDevOps/AzureDevopsSynchronizer.cs

```diff
@@ -237,13 +237,14 @@ private IEnumerable<WitBatchRequest> GetTestCasesToUpdate(Dictionary<int, WitBat
                 {
                     return false;
                 }
-                var fieldANormalized = Regex.Replace(fieldValue, @"\s", "").ToLowerInvariant();
-                var fieldBNormalized = Regex.Replace(dictionaryB[fieldKey], @"\s", "").ToLowerInvariant();
+
+                var fieldANormalized = Regex.Replace(fieldValue ?? string.Empty, @"\s", "").ToLowerInvariant();
+                var fieldBNormalized = Regex.Replace(dictionaryB[fieldKey] ?? string.Empty, @"\s", "").ToLowerInvariant();
```

Comment:
If whitespace replacement logic is the same for both fields maybe we should create a single method for them both and not double this two calls?

Reply:
Done